inertia/components/MatrixCard.vue

```diff
@@ -0,0 +1,90 @@
+<script lang="ts" setup>
+import type Matrix from '#models/matrix'
+import { router } from '@inertiajs/vue3'
+import Button from 'primevue/button'
+import Card from 'primevue/card'
+import { useConfirm } from 'primevue/useconfirm'
+import { onMounted, onUnmounted, ref } from 'vue'
+import { useTransmit } from '~/composables/use_transmit'
+
+const props = defineProps<{
+  matrix: Matrix
+}>()
+
+const gif = ref<string>('')
+const confirmDelete = useConfirm()
+
+const fetchGif = async () => {
+  const response = await fetch(`http://localhost:3333/matrices/${props.matrix.id}/render`)
+  const blob = await response.blob()
+  try {
+    const reader = new FileReader()
+    reader.onload = function () {
+      gif.value = reader.result as string
+    }
+    reader.readAsDataURL(blob)
+  } catch (e) {
+    console.error(e)
+  }
+}
+
+fetchGif()
+
+const transmit = useTransmit()
+
+const subscription = transmit.subscription(`matrix/${props.matrix.id}/render`)
+let stopListening: () => void
```

Comment:
assez étonné que typescript ne gueule pas à l'usage en bas (probablement parce qu'il est définit dans le onMounted, c'est pas très safe parce que si ca avait été un watch jamais trigger ca aurait été pareil)
```suggestion
let stopListening: (() => void) | undefined = undefined
```
Je pense qu'il vaut mieux être safe et faire ca